### ISSUE TITLE
Running `Import-VisualStudioVars` without argument fails if there are multiple `VS*COMNTOOLS` environment variables

### ISSUE THREAD

**Issue by i-give-up** [Pscx/Pscx#35]
I'm using Powershell version 4. I have 2 versions of Visual Studio (2008 Express and 2015 Build Tools) so I have both `VS140COMNTOOLS` and `VS90COMNTOOLS` environment variables.

When I attempt to run `Import-VisualStudioVars` without any argument, I get the error message

``` text
The variable '$matches' cannot be retrieved because it has not been set.
```

This is the part causing issue: https://github.com/Pscx/Pscx/blob/0444d473d3dd445ad7b5fd4fc7d46f47c5362483/Src/Pscx/Modules/Utility/Pscx.Utility.psm1#L2268-L2272

When the left hand size of `-match` is a list (as is the case when `Get-Item Env:\vs*comntools` returns multiple matches), it doesn't set the variable `$matches` at all which is what leads to this error.

I think a reasonable approach to get the latest version of Visual Studio if a version is not specified, though that would require some sorting (the default order would be for example `VS130COMNTOOLS`, `VS140COMNTOOLS`, `VS90COMNTOOLS` so that wouldn't work).

**Comment by rkeithhill** [comment]
Thanks for taking the time to file this issue.  Given that the "latest" version of Visual Studio does not define VS*COMNTOOLS environment variables **and** requires an additional module (VSSetup) to even find the install location, I'm inclined to suggest to folks that they pass in the version of VS they want as the first parameter.  And not rely on the "default" behavior.

It's tempting to mark the `VisualStudioVersion` parameter as mandatory (it probably should have been from the start) but that would be a breaking change.   Anyway, I can fix the above bug but I'm going to wait a bit for other bugs to filter in.

**Comment by rkeithhill** [comment]
This should be fixed in 3.3.1.
